python/py-svgpath/Portfile

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-svgpath
+version             6.3
+revision            0
+
+python.rootname     svg.path
+
+checksums           rmd160  bb83fad1fc8bd5132bf3fbf352cea2666c63d686 \
+                    sha256  e937740a316a7fec86acd217ab6226e112f51328078524126bb7ea9dbe7b1ade \
+                    size    20920
+
+maintainers         {@flyingsamson tuxcad.de:flyingsamson} openmaintainer
+description         SVG path objects and parser
+long_description    svg.path is a collection of objects that implement the different path \
+                    commands in SVG, and a parser for SVG path definitions.
+license             MIT
+
```

Comment:
this package appears to be only Python and nothing gets compiled. A good way to check is to check the download files on PyPI. In this case the .whl file contains "none-any". 
That means it installs the same files on all OS versions and is architecture independent. 

To state these things in the Portfile one would add:

supported_archs noarch
platforms {darwin any}

Reply:
Ah, thanks for pointing that out, and explaining what to look out for!